services/tenant-ui/frontend/src/router/index.ts

```diff
@@ -47,55 +40,51 @@ const routes = [
   },
 
   {
-    path: '/issuance/',
+    path: '/configuration/',
```

Comment:
do we want to move configuration down the menu? it's important but seems odd placed between contacts/issuer/verifier/holder... not important, just something that looked "off" to me.

Reply:
Yeah, the ticket didn't have any placement for it, but I'd agree, moving